core/templates/dev/head/css/oppia.css

```diff
@@ -119,6 +119,74 @@ body {
   margin-top: 15px;
 }
 
+.oppia-profile-tabs md-content {
+  background-color: white !important; 
+}
+
+.oppia-profile-tabs md-content md-tabs {
+  background: #f6f6f6;
+  border: 1px solid #e1e1e1; 
+}
+
+.oppia-profile-tabs md-content md-tabs md-tabs-wrapper {
+  background: white; 
+}
+
+.oppia-profile-tabs md-content h1:first-child {
+  margin-top: 0; 
+}
+
+.oppia-profile-tabs md-tab {
+  background-color: #009688;
+}
+
+.oppia-profile-tabs md-tab-ink-bar {
+  background-color: black;
+}
+
+
+.oppia-profile-tabs md-tab-label {
+  color: #ffffff;
+}
+
+.oppia-stat-container-top {
+  display: inline-block;
+  width: 45%;
+}
+
+.oppia-stat-container {
+  display: inline-block;
+  width: 25%;
+}
+
+.oppia-stat-big-container {
+  display: block;
+  margin-bottom: 50px;
+  text-align: center;
+  width: 100%;
+    
+}
+
+.oppia-big-top-container {
+  margin-bottom: 70px;
+}
+
+.oppia-stat-container-top span {
+  display: block;
+}
+
+.oppia-stat-container span {
+  display: block;
+}
+
+.oppia-big-number {
```

Comment:
added .oppia-stat-container in front of .oppia-big-number and .oppia-small-number